kubectl-socks5-proxy

```diff
@@ -6,6 +6,7 @@
 [[ -n $DEBUG ]] && set -x -e
 
 MAX_POD_CREATION_TIME=10  # unit: second
+DEFAULT_NAME=${USER}-proxy
```

Comment:
It looks like Pod name must consist of lower case alphanumeric characters or '-'. 
I came up with the following error when I tried with USER='yoichi.kawasaki'

```
* spec.containers[0].name: Invalid value: "yoichi.kawasaki-proxy": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```
So here is my suggestion for `DEFAULT_NAME` is this,
```suggestion
DEFAULT_NAME=$(echo ${USER}-proxy|sed -e 's/[^[:alnum:]|-]//g'|tr '[:upper:]' '[:lower:]')
```

Reply:
No problem, I added your suggestion.